recipes/vgraph/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set github_author = "bioinformed" %}
+{% set name = "vgraph" %}
+{% set version = "1.2.0" %}
+# held back until github archives work
+#{% set sha256 = "f727208daeb0a23e5248ddbe2fe2a3fdd8ca5e510cc4ce7aa6bbccb9b2bc5805" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  held back until github archives work
+  #fn: {{ name }}-{{ version }}.tar.gz
+  #url: http://github.com/pvanheus/{{ name }}/archive/v{{ version }}.tar.gz
+  #sha256: {{ sha256 }}
+  git_url: https://github.com/{{ github_author }}/{{ name }}.git
+  git_rev: v{{ version }} 
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python >3
+    - setuptools
+    - setuptools_scm 1.15.0
+    - setuptools_scm_git_archive 1.0
+    - cython
+    - pysam
+    - pytest
+    - pytest-runner
+    - coverage
+  run:
+    - python >3
```

Comment:
If this isn't `noarch: python`, `conda-build 2.1.x` will create defunct packages with this.
Just remove the ` >3`.